app/pages/CommodityCodePage.scala

```diff
@@ -16,12 +16,19 @@
 
 package pages
 
-import models.Commodity
+import models.{Commodity, UserAnswers}
 import play.api.libs.json.JsPath
 
+import scala.util.Try
+
 case object CommodityCodePage extends QuestionPage[Commodity] {
 
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "commodity"
+
```

Comment:
this was added because the HasCorrectGoods Page needed to have it's answer cleared each time the user went from the commoditycode page to HasCorrectGoods